### PR TITLE
Anonymise organisation specific server references in tests/dhcp

### PR DESCRIPTION
**Pull request by ndietsch**
There are server organisation specific server references in these files and I have anonymised them. Sorry for any noise created during the commit process.

### CODE REVIEW

File: test/dhcp/isc_omapi_provider_test.rb [hunk 1]
```diff
@@ -105,10 +105,10 @@ def test_sparc_host_quirks
     assert_equal [
       %q{option SUNW.JumpStart-server \"192.168.122.24:/Solaris/jumpstart\";},
       %q{option SUNW.install-path \"/Solaris/install/Solaris_5.10_sparc_hw0811\";},
-      %q{option SUNW.install-server-hostname \"itgsyddev807.macbank\";},
+      %q{option SUNW.install-server-hostname \"someserver.somedomain\";},
```
Comment (ekohl) [inline test/dhcp/isc_omapi_provider_test.rb:108]: It's common to use example.(com|org|net) for this.

**Review by ekohl — CHANGES_REQUESTED** [review]
Thanks! I'd noticed we had a few of these instances as well. Could you also squash your commits into one?

**Comment by ndietsch** [comment]
Hi,

 
I will resubmit it as a single example.com commit. 

 
Thanks,

 
Nathan 

-----Original message-----

**Comment by ndietsch** [comment]
Hi,

 
The pull request has been resubmitted as a single commit and I updated the references to example-server.example.com

 
Kind Regards,

 
Nathan Dietsch 

-----Original message-----
From: Nathan Dietsch
Sent: Sunday, August 18 2019, 6:19 pm
To: theforeman/smart-proxy
Subject: RE: [theforeman/smart-proxy] Anonymise organisation specific server references in tests/dhcp (#674)

Hi,

 
I will resubmit it as a single example.com commit. 

 
Thanks,

 
Nathan 

-----Original message-----

**Comment by lzap** [comment]
Ok you could have force-pushed here but this is fine.

**Pull request closed without merging**